### DISCUSSION TITLE
thanhle.blog/chiec-transaction-tren-blockchain-ben-trong-co-gi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#48] [Ideas]
# thanhle.blog/chiec-transaction-tren-blockchain-ben-trong-co-gi

Xem coi một transaction trong blockchain là gì? Có gì bên trong nó

https://thanhle.blog/blog/chiec-transaction-tren-blockchain-ben-trong-co-gi

**Comment by darkpiv** [comment]
nonce hiểu là gì hở e?

**Reply by thanhlmm** [reply]
Nonce (Number only used Once) dùng để định dang một transaction khi chưa được verify.
VD nếu gửi 2 transaction y chang nhau mà khác none, thì nghĩa là 2 tx (Chuyển tiền cho cùng 1 người, cùng 1 số tiền, 2 lần). Còn nếu cùng 1 nonce thì nó sẽ hiểu chỉ là một transaction mà bị duplicate request hoặc user muốn override/update lại transaction đó

Vd sử dụng nonce để speed up hoặc cancel tx https://metamask.zendesk.com/hc/en-us/articles/360015489251-How-to-speed-up-or-cancel-a-pending-transaction

**Comment by thanhlmm** [comment]
Bài viết khá hay "lan quyên"

https://www.twigblock.com/articles/eth-raw-data
